docs/run-azure.md

```diff
@@ -27,8 +27,8 @@ git push
 The pipeline will start creating an environment. Progress can be viewed here: [nhs-digital-gp-it-futures.platform-helm](https://buyingcatalog.visualstudio.com/Buying%20Catalogue/_build?definitionId=75&_a=summary)
 
 Once the environment is created, you'll see a namespace in the dashboard called `buyingcatalogue-<branch name>`.
-If you amend your hosts file as below (replacing <namespace>), you'll be able to browse to the environment:
 
+If you amend your hosts file as below (replacing <namespace>), you'll be able to browse to the environment:
```

Comment:
I was confused to see the original file had the <namespace> value in the brackets. Turns out markdown doesn't show this in the preview e.g. '(replacing ),'. I tried the value `replacing \<namespace>` & it rendered `replacing <namespace>`.